rotate.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# check if DOMAINKEY parameter is set, exit if not
+if [ -z "$DOMAINKEY" ]; then
+  echo "DOMAINKEY environment variable is not set"
+  exit 1
+fi
+
+DOMAINS=$@
+
+# if there are no domains, read from stdin and store in DOMAINS
+if [ -z "$DOMAINS" ]; then
+  DOMAINS=$(cat)
+fi
+
+# if there are still no domains, exit
+if [ -z "$DOMAINS" ]; then
+  echo "No domains specified. Add them as command line arguments or pipe through standard input."
+  exit 1
+fi
+
+echo 'domain, key'
```

Comment:
```suggestion
echo "domain, key"
```
For the sake of consistency